src/index.ts

```diff
@@ -82,17 +90,33 @@ function createSingleTicket(state: IClubhouseState) {
           return Promise.resolve(filtered);
         }
       },
+    },
+    {
+      type: 'list',
```

Comment:
There are three options here:
1) Instead of label support, add sprint support. Filter all labels and keep the ones that start with `Sprint`, parsing the dates and making the current one the default
2) Allow arbitrary label support, in which case the type should be checkbox not list, so a user can select multiple label
3) A hybrid approach. Change it to checkbox, but put the Sprint ones first, with the current one being the default

I prefer 1 or 3. The goal here is not to provide a generic CLI, but to make it easy and fast to create tickets.